sailthru/sailthru_client.py

```diff
@@ -539,6 +539,23 @@ def stats_blast(self, blast_id=None, start_date=None, end_date=None, options=Non
         data['stat'] = 'blast'
         return self._stats(data)
 
+    def stats_send(self, template=None, start_date=None, end_date=None, options=None):
```

Comment:
according to the docs, `stats/send` requires `template`, `start_date` and `end_date` - so I would not expect those to be defaulted